randovania/game_description/resources/resource_type.py

```diff
@@ -59,9 +59,6 @@ def non_negated_prefix(self) -> str:
         else:
             return ""
 
-    def __lt__(self, other):
```

Comment:
I take it this is rolled into `intEnum`?

Reply:
I imagine. I also tested in terminal and it worked just fine.